tests/test_predict.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import pytest
+
+from context import predicting_jams
+from predicting_jams.predict import *
```

Comment:
\[Pylint\] ERROR: Unable to import 'predicting\_jams\.predict' \[import\-error\]